src/components/editMeasure/populationCriteria/PopulationCriteriaHome.tsx

```diff
@@ -45,11 +45,11 @@ export function PopulationCriteriaHome() {
   })();
 
   useEffect(() => {
-    if (path.includes("/groups")) {
+    if (pathname.includes("/groups")) {
       if (+groupNumber && +groupNumber > 0) {
         setMeasureGroupNumber(+groupNumber - 1);
       } else {
-        history.push("/404");
+        // navigate("/404");
```

Comment:
Is this block of code unreachable ?

Reply:
I don't think so. I would think it would be 404'd before ever reaching this, but I'll uncomment in case.